app/javascript/pages/admin/IndividualEvents/main.css

```diff
@@ -178,3 +178,20 @@ textarea {
 .muiTextField hr {
   display: none;
 }
+
+.eventsTable > div {
+  border: none;
+}
+
+ .eventsTable > div > div:nth-child(1) > div:nth-child(1) {
```

Comment:
same as above comment. I'm not sure what `eventsTable>div[1][1]` means at a glance